lib/ex_ice/ice_agent.ex

```diff
@@ -119,6 +119,22 @@ defmodule ExICE.ICEAgent do
     GenServer.call(ice_agent, :get_local_credentials)
   end
 
+  @doc """
+  Gets local candidates that have been already gathered.
+  """
+  @spec get_local_candidates(pid()) :: [binary()]
```

Comment:
I would use `String.t()`?

Reply:
I used the same type as in `signal()` with `{:new_candidate, binary()}`, I would rather keep it consistent and change both or none.